test.js

```diff
@@ -447,3 +450,60 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test.cb('e2e', t => {
+  const id = uid(16)
+
+  const testFn = function () {
```

Comment:
If you make this an `async function`, you’ll be able to `await` promises.

Reply:
I don't think this project is setup for async/await https://github.com/segmentio/analytics-node/pull/134#issuecomment-344448401.

I tried:

```
const response = await axios.get(....)
```

And got the error: "Parsing error: The keyword 'await' is reserved".

Reply:
Right, you need to make the `testFn` async:

```js
const testFn = async function () {
  await somePromise()
}
```

Reply:
better:

```js
const testFn = async () => {

}
```

Reply:
Fixed now.